src/app/modules/submission/email-submission/email-submission.middleware.ts

```diff
@@ -64,8 +64,8 @@ export const validateResponseParams = celebrate({
             fieldType: Joi.string()
               .required()
               .valid(...Object.values(BasicField)),
-            answer: Joi.string().allow(''),
-            answerArray: Joi.array(),
+            answer: Joi.string().allow('').insensitive(),
+            answerArray: Joi.array().items(Joi.string().insensitive()),
```

Comment:
nit: I don't think the `.insensitive()` part is necessary, since we don't restrict the types of strings we allow